manifest/provider/import_test.go

```diff
@@ -50,15 +50,15 @@ func TestParseImportID(t *testing.T) {
 			t.Fatal(actualErr.Error())
 		}
 		if expected.GVK != actualGvk {
-			t.Log("GVK (actual / wanted):", actualGvk, expected.GVK)
+			t.Logf("GVK did not match. actual: %q  expected: %q.", actualGvk, expected.GVK)
```

Comment:
Took me a minute to figure out which case was actually failing so I made this a bit more readable.